btr-web/btr-main-app/components/person-info/Details.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="information">
+    <!-- NB: if the UIcons are not wrapped in dev they render at differring sizes -->
```

Comment:
I would potentially put` class="w-x h-y" ` if you have specific size they can be